wiki/Skinning/osu!taiko/en.md

```diff
@@ -1,528 +1,526 @@
-[true]: /wiki/shared/true.png

```

Comment:
can you split the line endings into a different commit? it's hard to tell what changes were made

Reply:
Waiting until the other PR gets merged.